src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -157,6 +157,7 @@ public interface IHtmlGenerator
             ModelExplorer modelExplorer,
             string expression,
             object value,
+            string format,
```

Comment:
The easiest way is comes into my mind feels like a hack. But, (for 1.1.0) release may be we should call `IHtmlGenerator.GenerateInput` instead of `IHtmlGenerator.GenerateHidden`. We'll duplicate 2 lines of serialization code that way, but nothing else will be affected.